plugins/inputs/opcua/opcua_client_test.go

```diff
@@ -20,7 +20,6 @@ type OPCTags struct {
 }
 
 func TestClient1Integration(t *testing.T) {
-	t.Skip("Skipping due to dial tcp 195.254.227.245:4840: connect: connection refused")
```

Comment:
Could you add in --> 	

```
if testing.Short() {
		t.Skip("Skipping integration test in short mode")
	}
```

Otherwise it will fail on circleci as it's trying to run it and we don't currently have our build process setup to run integration tests with the docker file

Reply:
Removed the keyword and added the short test exemption.